### DISCUSSION TITLE
Raindrop.io 加星后自动导入到简悦并生成本地快照

### DISCUSSION THREAD

**Discussion by Kenshin** [Kenshin/simpread#3021] [服务]
> 简悦 + Raindrop.io 的方式可以弥补 Raindrop.io 基础版账户无法实现收藏文章快照的缺陷。

https://user-images.githubusercontent.com/81074/140641859-07345388-c93d-4a91-8cdf-5291546ccbc9.mp4

> 如果你喜欢使用书签服务的话，我推荐你使用 [Raindrop.io](https://raindrop.io/)，这是一个老牌的书签服务了，在制作简悦前我在这里已经收藏了 12xx 个书签。

![image](https://user-images.githubusercontent.com/81074/140635714-07b77c89-b574-4eb2-9eec-4f72b4dc4c82.png)

![image](https://user-images.githubusercontent.com/81074/140635729-7ebccb5d-7209-4a74-a77f-3646394267a5.png)

# 优势

> 相比其它服务来说，它的优势很明显

- 从 2015 年就存在至今，完全可以自给自足。
- 国外服务，不会出现收藏之后就无故消失的问题。
- 无限的书签 / 合辑

> 缺点

- 当收藏的内容为文章时，无法保存快照

# 为什么我会推使用 Raindrop.io

> 作为一个书签工具或者说一个「收藏」工具来说，最重要的几点：

- [x] 尽可能长时间的存在，一个 15 年的工具存活至今，足以说明。
- [x] 免费版足够使用（ Raindrop.io 的免费版本支持无限个书签）
- [ ] 收录的文章尽可能的快照功能。

显然，Raindrop.io 除了第三点外，其余完全满足需求，但你可以使用**简悦来弥补第三点的不足**，即：

> 收藏的文章加星后，会自动出现在**简悦的稍后读**以及[**自动保存快照**](https://github.com/Kenshin/simpread/discussions/2994)。

# 同属与「稍后读」为什么简悦并不排斥同类产品

> 如果你是 Hypothesis 的用户，自然知道简悦也可以 [导入到 Hypothesis](https://github.com/Kenshin/simpread/discussions/2085#discussioncomment-926749)。

> 其根本原因是：简悦是真正意义的不绑架用户数据，所以**简悦并不会像其它应用一样，限制用户使用简悦的方式**。

# 前置知识

> 需要事先完成下面的配置。

1. [如何开通开放平台](https://github.com/Kenshin/simpread/discussions/2757)
2. [API 快照功能](https://github.com/Kenshin/simpread/discussions/2994)
3. [如何让稍后读支持读取本地缓存](https://github.com/Kenshin/simpread/discussions/2146)

# 前置知识（二）

1. 此方式需要使用 Zapier 这个自动化工具，如果你不了解它的话，可以类比是 IFTTT，但比 IFTTT 提供更强大 / 灵活的自动化定制。

2. 你可以使用此 [邀请链接](https://zapier.com/developer/public-invite/121575/cbaa1ac9bb7fa64f024515b7f46d5e1b/) 来自动化配置 Raindrop.io 与简悦。

3. 简悦需要开通开放平台服务，如何开通 https://github.com/Kenshin/simpread/discussions/2757

# 如何配置

> 如果你使用过 Zapier，整个过程不会超过五分钟，否则请根据下图的一步步操作即可。

![image](https://user-images.githubusercontent.com/81074/140633810-bd7e2d70-5890-4e43-a4a1-0f589eadcd82.png)

![image](https://user-images.githubusercontent.com/81074/140633824-a3e5a982-1c59-47d9-be1e-84c1c66dce7f.png)

![image](https://user-images.githubusercontent.com/81074/140633843-1ef5de13-6079-4330-b168-aeefe02c23f8.png)

![image](https://user-images.githubusercontent.com/81074/140633863-472e8311-9aa6-49c9-80ec-cc889ab54505.png)

![image](https://user-images.githubusercontent.com/81074/140633880-f337faff-fcf1-494c-a46c-3d550b711c63.png)

![image](https://user-images.githubusercontent.com/81074/140633897-634e4f9b-5e67-4c62-8008-d7d9fe9aad47.png)

![image](https://user-images.githubusercontent.com/81074/140634168-5a413d10-e20c-4a41-b1f0-ec47c049bbb4.png)

> 注意：下图是 **1.0.0** 版的截图，请使用 **1.0.1** 版本

![image](https://user-images.githubusercontent.com/81074/140634002-40667961-cd8d-43fc-948e-57119efe7317.png)

![image](https://user-images.githubusercontent.com/81074/140634190-95637710-4223-4de4-b0a6-376a41cdbb8f.png)

![image](https://user-images.githubusercontent.com/81074/140634218-d2d99a80-bf9e-483a-8809-2ce03dbeedc3.png)

![image](https://user-images.githubusercontent.com/81074/140634251-0c0aac48-1708-487f-98be-662d2454a229.png)

![image](https://user-images.githubusercontent.com/81074/140634265-447f066a-8db3-4654-9254-d40aa8a572fc.png)

![image](https://user-images.githubusercontent.com/81074/140634282-e52e1960-795e-4989-8df1-76ffabea5255.png)

![image](https://user-images.githubusercontent.com/81074/140634302-0e307204-d350-41f5-9b94-c7dc4103394c.png)

# 关联

- [利用 Airtable 实现属于自己的链接公开分享集 #3003](https://github.com/Kenshin/simpread/discussions/3003)
- [加入 Pocket 后自动保存到简悦的稍后读（包含快照） #3024](https://github.com/Kenshin/simpread/discussions/3024)
- [Instapaper 加星后自动保存到简悦的稍后读（包含快照） #3034](https://github.com/Kenshin/simpread/discussions/3034)
- [将简悦变成你的 RSS 阅读器（自动下载快照到本地） #3036](https://github.com/Kenshin/simpread/discussions/3036)
- [保存到 Pinboard.in 同时也保存到简悦（包括本地快照） #3048](https://github.com/Kenshin/simpread/discussions/3048)
- [Inoreader 加星后自动保存到简悦的稍后读以及生成本地快照 #3007](https://github.com/Kenshin/simpread/discussions/3007)
- [订阅 Newsletter 后自动保存到简悦的稍后读 #3088](https://github.com/Kenshin/simpread/discussions/3088)
- [当加入稍后读后，自动发送到你的 Telegram Channel #3092](https://github.com/Kenshin/simpread/discussions/3092)
- [发送当前 URL 到飞书群（包括：加入稍后读或手动触发）#3104](https://github.com/Kenshin/simpread/discussions/3104)
- [保存 RSS 的全文到简悦的稍后读（包括快照） #3186](https://github.com/Kenshin/simpread/discussions/3186)

# 引申

- [[教程\]如何开通开放平台 #2757](https://github.com/Kenshin/simpread/discussions/2757)
- [当在开放平台设置完毕后，如何测试配置是否成功？ #3020](https://github.com/Kenshin/simpread/discussions/3020)
- [授权登录（如何使用第三方登录简悦的高级账户） #1570](https://github.com/Kenshin/simpread/discussions/1570)
- [个人中心 #2959](https://github.com/Kenshin/simpread/discussions/2959)
- [免密码登录 #2974](https://github.com/Kenshin/simpread/discussions/2974)

**Comment by Lapis0x0** [comment]
好耶，照着教程做成功了！

**Reply by Kenshin** [reply]
哈哈，是新用户吗？恭喜你 Level 直接满级！

PS：请问是 Raindrop.io 重度使用者吗？如果是的话，能否来个工作流，我放到 https://github.com/Kenshin/simpread/discussions/2085 这里。

**Comment by HeatDeathOfUniverse** [comment]
自动化全部是绿勾，但是我加“最喜爱的”好像没用？添加到稍后读却可以自动下载到本地，我用的Dropbox

**Reply by Kenshin** [reply]
调用 API 是有个时间差的，另外你给出的信息太少了，我也没法判断。

建议详细描述后，发到 https://github.com/kenshin/simpread/issues/new 这里。

**Comment by vicentory** [comment]
找不到simpread应用啊，也看不到图片里的邀请链接。
<img width="535" alt="image" src="https://user-images.githubusercontent.com/10493597/153126926-416e85fe-c65a-4a13-b769-74142dffd655.png">

**Reply by Kenshin** [reply]
邀请链接 https://zapier.com/developer/public-invite/121575/cbaa1ac9bb7fa64f024515b7f46d5e1b/

教程里面当然会包含邀请链接了

![image](https://user-images.githubusercontent.com/81074/153127337-86c03618-e014-4e99-84d4-62dd3d9b7751.png)
